cmp/grid/impl/ColumnHeader.ts

```diff
@@ -146,15 +154,19 @@ class ColumnHeaderModel extends HoistModel {
     get gridModel() {
         return this.componentProps.gridModel;
     }
-    get xhColumn() {
+
+    get xhColumn(): Column {
         return this.componentProps.xhColumn;
     }
+
     get agColumn() {
         return this.componentProps.column;
     }
+
     get colId() {
         return this.agColumn.colId;
```

Comment:
Should this be `getColId()`? Just checking since I saw it was changed in several other places.

Reply:
fixed -- thanks